avocado/core/remote/runner.py

```diff
@@ -264,6 +269,10 @@ def run_suite(self, test_suite, variants, timeout=0, replay_map=None):
             self.result.end_tests()
             self.job._result_events_dispatcher.map_method('post_tests',
                                                           self.job)
+            # compare the major version of remote and local version of avocado. If different write to UI and job.log
+            if str(remote_version[0]) != version.MAJOR:
+                APP_LOG.warn(warning_message, '.'.join(map(str, remote_version)), version.VERSION)
+                TEST_LOG.warn(warning_message, '.'.join(map(str, remote_version)), version.VERSION)
```

Comment:
Yup I can do that, I want to make sure I understand though. Instead of calling `join/map` twice just make one call to `join/map` that will update the `warning_message` variable and write that out, is that correct?